models/payment/src/payment-status/payment-status-draft/generator.ts

```diff
@@ -0,0 +1,14 @@
+import { Generator } from '@commercetools-test-data/core';
+import { TPaymentStatusDraft } from '../types';
+
+// https://docs.commercetools.com/api/projects/payments#paymentstatusdraft
+
+const generator = Generator<TPaymentStatusDraft>({
+  fields: {
+    interfaceCode: null,
+    interfaceText: null,
+    state: null,
```

Comment:
Here we can safely use the `Reference` since both REST and GraphQL expect it.

```suggestion
    state: fake(() => Reference.random().typeId('state')),
```